integrations/spark/spark-job-dataquality/spark-job-dataquality-app/src/main/java/com/thinkbiganalytics/spark/dataquality/rule/InvalidRowPercentRuleImpl.java

```
@@ -0,0 +1,131 @@
+package com.thinkbiganalytics.spark.dataquality.rule;
+
+/*-
+ * #%L
+ * kylo-spark-job-dataquality-app
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Yes, you can ignore the errors related to the file header.